index.js

```diff
@@ -45,6 +46,20 @@ function handleArgs(cmd, args, opts) {
 	};
 }
 
+function handleInput(spawned, opts) {
+	var input = opts.input;
+
+	if (input === null || input === undefined) {
+		return;
+	}
+
+	if (isStream(input)) {
+		input.pipe(spawned.stdin);
+	} else {
+		spawned.stdin.end(input);
+	}
```

Comment:
Nah, I guess it's a personal preference. If I can, I skip `else` and `return` early. To me at least, it's somewhat clearer.